cmd/cloud/organization.go

```diff
@@ -54,6 +62,24 @@ func newOrganizationSwitchCmd(out io.Writer) *cobra.Command {
 	return cmd
 }
 
+func newOrganizationExportAuditLogs(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "export-audit-logs [organization_name] --output-file [output_file.json]",
```

Comment:
```suggestion
		Use:     "export-audit-logs [organization_name] --output-file [output_file.json]",
```
General command structure of most CLI commands has been `astro <resource> <action>` or `astro <resource> <subresource> <action>` should we try and adhere to that in order to keep things consistent? cc @kushalmalani @sunkickr

Also, we have been using `organization id` for switch command, so it would be ideal from user POV to be consistent and use id over name. 

maybe final command structure could be something like: `astro organization logs export --organization-id=< organization_id>` or `astro organization audit-logs export --organization-id=< organization_id>` because generally the argument is for the resource identifier. In this case, the resource is the log and the organization is more of a parent resource from the CLI command point of view.

Reply:
Let's do `astro organization audit-logs export` then. I think this might fit better the future where we can create/update/delete orgs from the CLI?

Reply:
Hmm, in that case isn't there any API on the Astrohub API side to get the organization name from the organization id? If not then might want to at least take the organization name as a flag value instead of an argument to the command, since that's the parent resource identifier to be consistent with other command structure, i.e. `astro organization audit-logs export --organization-name=[organization_name]`

Reply:
@maxdml Does the user need to pass the organization name...If they are already logged in, the cli knows what org they are logged into. We can just get that info from context and find the orgName for it and send in api call rather than asking the user to input it.

`astro organization audit logs export <path>`, if no path is specified, we just default to export it to current directory

Reply:
My preference would be to avoid network calls if possible. Ideally the context would hold onto both ID and org name.

All organization-related core API endpoints take the organization short name in the `path`. We would essentially be adding one more network roundtrip + processing at the API for each of these commands. For short commands this can add a noticeable delay in the CLI responsiveness. In addition, eventually this will add unnecessary load to the API.

Reply:
Makes sense to avoid an api call. If we can save the organization name to context, then let's just re-use that rather than asking the user to input it.

Reply:
I think adding the name to the context should be in a different PR (I'm not super familiar with that part of the code yet).